### ISSUE TITLE
Account-level defaults for /cookie_sync

### ISSUE THREAD

**Issue by bretg** [prebid/prebid-server#1641]
As a followup to https://github.com/prebid/prebid-server/issues/924, there's a request to add server-side per-account control for the coopSync and limit flags -- being able to specify the default values.

The proposal is to add (yet another) column to the DB. `endpoint_config` column with JSON type.

We'd support 3 values initially:
```
{
 "cookie_sync": {
    "defaultLimit": 4,
    "maxLimit": 8,
    "defaultCoopSync": false
   }
}
```
Meanings:
- cookie_sync.defaultLimit: if the "limit" isn't specified in the /cookie_sync request, this is what to use
- cookie_sync.maxLimit: if the "limit" is specified, it can't be greater than this value
- cookie_sync.defaultCoopSync: if the "coopSync" value isn't specified in the /cookie_sync request, use this

**Comment by SyntaxNode** [comment]
Why do publishers need PBS to store default values instead of passing it with the cookie sync request as we have today?

**Comment by bretg** [comment]
This could be used in the future for preference of iframe-vs-redirect syncs when that feature is built.

**Comment by SyntaxNode** [comment]
Implemented in PBS-Go 0.204.0.
